src/models/post/stream.rs

```diff
@@ -57,36 +40,37 @@ impl PostStream {
     }
 
     pub async fn get_posts(
-        post_stream_values: PostStreamValues,
-        post_stream_filters: PostStreamFilters,
+        stream_params: PostStreamQuery,
```

Comment:
There is no need to feed PostStreamQuery to `get_posts`

We are now doing something different in this endpoint versus every other one.

By doing this we: can't have optional `source` and default to `All`. let's just pass down the components: pagination, source, sorting, etc.